test/anacondalib.py

```diff
@@ -60,6 +77,30 @@ def setUp(self):
             self.allow_browser_errors(".*client closed.*")
             self.allow_browser_errors(".*Server has closed the connection.*")
 
+    def _execute(self, cmd):
+        return subprocess.check_call(cmd, stderr=subprocess.STDOUT, shell=True)
+
+    def add_disk(self, size, backing_file=None):
+        image = self._create_disk_image(size, backing_file=backing_file)
+        self._execute(f"virt-xml -c qemu:///session {self.machine.label} --update --add-device --disk {image},format=qcow2,size={size}")
+
+        if self.is_nondestructive():
+            self.addCleanup(self.rem_disk, image)
+
+        return image
+
+    def rem_disk(self, disk):
+        self._execute(f"virt-xml -c qemu:///session {self.machine.label} --update --remove-device --disk {disk}")
+        self._execute(f"rm -f {disk}")
+
+    def _create_disk_image(self, size, image_path=None, quiet=False, backing_file=None):
+        if not image_path:
+            _, image_path = tempfile.mkstemp(suffix='.qcow2', prefix=f"disk-anaconda-{self.machine.label}")
```

Comment:
The directory defaults to /tmp, you need to be a bit careful as that's a tmpfs and easily overflows (or at least unnecessarily eats lots of RAM). This is the kind of "large file" that belongs into `dir="/var/tmp"`.

Reply:
Fixed.